src/web/src/utils.js

```diff
@@ -294,3 +301,28 @@ export const exportScheduleToIcs = (selectedCourses) => {
     })}_Schedule`
   );
 };
+
+/**
+ * Export All Selected Course Sections To An Image Format, Probably PNG.
+ * Can Come Back Here + Add More File Format Options, Such As JPEG, ...
+ */
+export const exportScheduleToImage = (selectedCourses) => {
+  // Handle Special Case Where No Selected Courses On Schedule.
+  if(selectedCourses.length == 0){
+    alert("No Courses Found For Export To Image Data.")
+    return;
+  }
+  // Obtain Schedule Element Defined In Schedule.Vue File + Run Export To PNG. 
+  var currentScheduleNode = document.getElementById('allScheduleData');
+  domtoimage.toPng(currentScheduleNode)
+    .then(function (dataUrl) {
+        var link = document.createElement('a');
+        link.download = 'allScheduleData.png';
```

Comment:
Let's name this like yacs-{{semester}}.png
I.e. yacs-spring-2021.png for this semester

This way people can more easily search for the images